Out sql statement

dora56 opened the pull request:
## 概要

srcフォルダでの@mikegami のファイル統合とプロジェクトフォルダの整理
## 技術的変更点

主な変更点は
- 822d9ff strage.js function名を変更 clickイベントでSQL文生成、strage格納
- 7665be6 プロジェクトフォルダの整理 @mikegami のファイルをsrcにコピー
- 5820603 CREATEページでsql文を表示
- 6a3507c publicフォルダへの展開を修正
## 注意事項
- コピーした`SQLcreate.html`を`index.html`にリネームしています。<br> 11df6c5 のコミットメッセージを確認してください
- publicフォルダを展開して簡易的な動作確認はしました。各自、pullして動作確認をお願いします。
- margeされた場合、createでの動作確認を http://ASO-SQLGenerator.github.io/ で行えるようになります。
## チェックリスト
- [x] CREATE文がで表示されるか
- [x] JSONがlocaStrageに格納されているか
- [x] テーブルの表示ができているか
## その他

現状ファイルの整合性がとれていないので、srcフォルダへの反映の仕方を話し合う必要があると思います。

dora56 committed:
不要なファイルの整理

bundle時にでエラーが出ないようにするため大量削除
drawing.jsはエラーが出るため一時的に削除、原因調査中

dora56 committed:
コード規約の変更

jqueryのコードを有効化
Handsontableのグローバル変数有効化

dora56 committed:
スクリプト追加とファイルの整形

dora56 committed:
SQL文の表示

htmlでは、テーブルが表示されてない状態
SQL文はsessionStorageに格納する。

mikegami committed:
cherry pick d8c20fa

dora56 committed:
プロジェクトフォルダの整理

- 直下にあった不要なフォルダの削除
- srcにcss、vendor、SQLcreate.htmlなどを追加

dora56 committed:
SQL文の表示

追加したhtmlがSQL文を表示できないため、TextAreaを追加
また、id要素で表示するSQLを分けるようにした

dora56 committed:
デプロイ用のタスクの修正

scrフォルダにvendorフォルダを追加したため、その分のコピーが必要であった

dora56 committed:
SQLcreate.htmlからindex.htmlにリネーム

ASO-SQLGenerator.github.ioのアクセス簡略化のため

SQLcreate.htmlのままではURLがhttp://ASO-SQLGenerator.github.io/SQLcreate.html
としなければならないため、http://ASO-SQLGenerator.github.io/　とアクセスできるように
index.htmlとした。

dora56 committed:
コピーではなく移動をしていたのを修正

friendly5024 commented:
deleteの動作がおかしい件の直し方。
drowing.jsをbundle.jsに含めないようにvendorフォルダに移動。
index.htmlにbundle.jsの読み込み後に
<script type="text/javascript" src="vendor/drawing.js"></script>　追加でなおるはず。
チェックリストの件は問題無し。

Commit:
deleteの挙動がおかしいのを直しました

dora56 committed:
Merge branch 'master' into outSqlStatement

Conflicts:
	ikegamidayo/src/js/drawing.js

dora56 committed:
masterの変更分を反映

dora56 committed:
指定していない制約が表示されるバグを修正

Columnを２つ以上指定していて、Aの制約が2つ以上、BがAの制約以下のときに
Bの制約にAの制約が混入するバグを修正

原因:
制約を格納する配列が初期化されないで、そのまま利用していたため

dora56 commented:
masterの変更点の反映と制約のバグの修正をしました。

**TODO**
- [x] `vender.js`をどうさがおかしいところを修正してバンドルに格納する。

dora56 committed:
`drawing.js` の動作修正

クリックした時に内部関数が呼ばれている状態になっていたので、
`tableDelete`関数をglobal関数に指定することで一時的に対応

dora56 commented:
`tableDelete`関数をglobalに呼び出しているので後で内部的に呼び出すようにする必要があります。
